### PR TITLE
Fission output type blog post

### PR DESCRIPTION
**Pull request by gauravgahlot**
Signed-off-by: Gaurav Gahlot <gauravgahlot0107@gmail.com>

**What type of PR is this?**

/kind content

**Any specific area of the project related to this PR?**

/area blog

### CODE REVIEW

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+> *This blog post is part of a series of articles about how to create a `Kubernetes` response engine with `Falco`, `Falcosidekick` and a `FaaS`.*
+> 
+> See other posts:
+> * [Kubernetes Response Engine, Part 1 : Falcosidekick + Kubeless]({{< ref "/blog/falcosidekick-reponse-engine-part-1-kubeless" >}})
+> * [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})
+> * [Kubernetes Response Engine, Part 3 : Falcosidekick + Knative]({{< ref "/blog/falcosidekick-reponse-engine-part-3-knative" >}})
+> * [Kubernetes Response Engine, Part 4 : Falcosidekick + Tekton]({{< ref "/blog/falcosidekick-reponse-engine-part-4-tekton" >}})
+> * [Kubernetes Response Engine, Part 5 : Falcosidekick + Argo]({{< ref "/blog/falcosidekick-reponse-engine-part-5-argo" >}})
+> * [Kubernetes Response Engine, Part 6 : Falcosidekick + Cloud Run]({{< ref "/blog/falcosidekick-reponse-engine-part-6-cloud-run" >}})
+> * [Kubernetes Response Engine, Part 7: Falcosidekick + Cloud Functions]({{< ref "/blog/falcosidekick-reponse-engine-part-7-cloud-functions" >}})
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:17]:
```suggestion
> * [Kubernetes Response Engine, Part 7 : Falcosidekick + Cloud Functions]({{< ref "/blog/falcosidekick-reponse-engine-part-7-cloud-functions" >}})
```

**Comment by Issif** [comment]
/hold

We can't release this post before next version of falcosidekick

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+We need tools with the following minimum versions to achieve this demo:
+
+* Minikube v1.19.0
+* Helm v3.5.4
+* kubectl v1.21.0
+* fission-cli v1.13.1
+
+### Provision local Kubernetes Cluster
+
+There are various ways to provision a local Kubernetes cluster such as, KinD, k3s, k0s, Minikube etc. We are going to
```
Comment (leogr) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:38]:
```suggestion
There are various ways to provision a local Kubernetes cluster such as KinD, k3s, k0s, Minikube, etc. We are going to
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 3]
```diff
@@ -0,0 +1,505 @@
...
+> * [Kubernetes Response Engine, Part 5 : Falcosidekick + Argo]({{< ref "/blog/falcosidekick-reponse-engine-part-5-argo" >}})
+> * [Kubernetes Response Engine, Part 6 : Falcosidekick + Cloud Run]({{< ref "/blog/falcosidekick-reponse-engine-part-6-cloud-run" >}})
+> * [Kubernetes Response Engine, Part 7 : Falcosidekick + Cloud Functions]({{< ref "/blog/falcosidekick-reponse-engine-part-7-cloud-functions" >}})
+
+----
+
+The earlier posts in this series, show how to use Kubeless, Argo, Knative, and others to trigger a resource after getting input from Falcosidekick.
+Recently, Falcosidekick received a new output type support for [Fission](https://github.com/falcosecurity/falcosidekick/pull/255).
+
+In this blog post, we will cover using `Falcosidekick` and `Fission` to detect and delete a compromised pod in Kubernetes cluster.
```
Comment (leogr) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:24]:
```suggestion
In this blog post, we will cover using `Falcosidekick` and `Fission` to detect and delete a compromised pod in a Kubernetes cluster.
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 4]
```diff
@@ -0,0 +1,505 @@
...
+
+Let's get the function and other artifacts:
+
+```
+$ git clone https://github.com/fission/examples.git && cd examples/sample/falco
+```
+
+The function we are going to deploy basically receives events for an infected pod from the _Falcosidekick_ and deletes it immediately.
+Before deploying the function we need some permissions to delete Pod.
+We create a `ServiceAccount` with right to delete a Pod in any namespace, and we'll associate it to our function:
```
Comment (leogr) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:248]:
```suggestion
We create a `ServiceAccount` with rights to delete a Pod in any namespace, and we'll associate it to our function:
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 5]
```diff
@@ -0,0 +1,505 @@
...
+```shell
+$ fission function logs -f --name falco-pod-delete
+..
+[2021-07-21 10:47:27.206605532 +0000 UTC] 2021/07/21 10:47:27 Deleting pod alpine from namespace default
+..
+```
+
+## Conclusion
+
+With this really simple example, we got another way to create a Response Engine with amazing pieces of software from Open Source world.
```
Comment (leogr) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:491]:
```suggestion
With this really simple example, we got another way to create a Response Engine with amazing pieces of software from the Open Source world.
```

**Review by leogr — COMMENTED** [review]
Just found some minor issues, otherwise looks good to me! :clap:

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 6]
```diff
@@ -0,0 +1,505 @@
+---
+title: "Kubernetes Response Engine, Part 8: Falcosidekick + Fission"
+date: 2021-07-22
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:3]: Please update the date to a more recent one (after release of 2.24.0 of falcosidekick)

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 7]
```diff
@@ -0,0 +1,505 @@
+---
+title: "Kubernetes Response Engine, Part 8: Falcosidekick + Fission"
+date: 2021-08-19
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:3]:
```suggestion
date: 2021-08-29
```

**Review by Issif — CHANGES_REQUESTED** [review]
please update the date of the post

**Review by Issif — APPROVED** [review]

**Review by leogr — APPROVED** [review]
LGTM

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 8]
```diff
@@ -0,0 +1,505 @@
+---
+title: "Kubernetes Response Engine, Part 8: Falcosidekick + Fission"
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:2]:
```suggestion
title: "Kubernetes Response Engine, Part 9: Falcosidekick + Fission"
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 9]
```diff
@@ -0,0 +1,505 @@
...
+date: 2021-08-29
+author: Gaurav Gahlot
+slug: falcosidekick-reponse-engine-part-8-fission
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:5]:
```suggestion
slug: falcosidekick-reponse-engine-part-9-fission
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md [hunk 10]
```diff
@@ -0,0 +1,505 @@
...
+> *This blog post is part of a series of articles about how to create a `Kubernetes` response engine with `Falco`, `Falcosidekick` and a `FaaS`.*
+> 
+> See other posts:
+> * [Kubernetes Response Engine, Part 1 : Falcosidekick + Kubeless]({{< ref "/blog/falcosidekick-reponse-engine-part-1-kubeless" >}})
+> * [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})
+> * [Kubernetes Response Engine, Part 3 : Falcosidekick + Knative]({{< ref "/blog/falcosidekick-reponse-engine-part-3-knative" >}})
+> * [Kubernetes Response Engine, Part 4 : Falcosidekick + Tekton]({{< ref "/blog/falcosidekick-reponse-engine-part-4-tekton" >}})
+> * [Kubernetes Response Engine, Part 5 : Falcosidekick + Argo]({{< ref "/blog/falcosidekick-reponse-engine-part-5-argo" >}})
+> * [Kubernetes Response Engine, Part 6 : Falcosidekick + Cloud Run]({{< ref "/blog/falcosidekick-reponse-engine-part-6-cloud-run" >}})
+> * [Kubernetes Response Engine, Part 7 : Falcosidekick + Cloud Functions]({{< ref "/blog/falcosidekick-reponse-engine-part-7-cloud-functions" >}})
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:17]: Can you add the part 8 with Flux v2 please

**Review by Issif — CHANGES_REQUESTED** [review]

**Review by Issif — APPROVED** [review]

**Comment by gauravgahlot** [comment]
Needs a rebase after #492 is merged.

**Review by leogr — APPROVED** [review]

**Pull request merged**